deploy/deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
`hack/deploy.sh` already covers most of this.  You just need to remove the component CR creation on lines 50, 55, and 60.

Reply:
Another issue when just running hack/deploy by itself (not part of a make target) is that _out is empty. The new deploy can be used on any cluster, not created by a hco make.